steps/dialog-notifier/0.1.0/step.yml

```diff
@@ -0,0 +1,42 @@
+title: Dialog Notifier
+summary: |
+  Notify Dialog chat about build status
+description: |
+  Notify Dialog chat about build status
+website: https://github.com/dialogs/bitrise-step-dialog-notifier
+source_code_url: https://github.com/dialogs/bitrise-step-dialog-notifier
+support_url: https://github.com/dialogs/bitrise-step-dialog-notifier/issues
+published_at: 2019-08-29T13:28:07.135529+03:00
+source:
+  git: git@github.com:dialogs/bitrise-step-dialog-notifier.git
+  commit: 3d45e3e2b5eda44e4210d930235d52b499d30c3a
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+- notification
+toolkit:
+  go:
+    package_name: github.com/dialogs/bitrise-step-dialog-notifier
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: true
+is_skippable: false
+run_if: ""
+inputs:
+- DIALOG_WEBHOOK_URL: $DIALOG_WEBHOOK_URL
```

Comment:
Inputs should use lowercase letter. please check the step development guide:
https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md